packages/office-ui-fabric-react/src/components/FocusZone/FocusZone.types.ts

```diff
@@ -156,7 +162,13 @@ export interface IFocusZoneProps extends React.HTMLAttributes<HTMLElement | Focu
    */
   onFocusNotification?: () => void;
 }
-
+/**
+ * State for the focusZone component
+ */
+export interface IFocusZoneStates {
```

Comment:
Can we make this `IFocusZoneState`? Just to follow the naming convention we have in other controls in the library :)

Reply:
sure :)